app/core/user/user.service.ts

```diff
@@ -132,10 +133,53 @@ export class UserService {
     data += 'FullName=' + user.firstName + ' ' + user.lastName;
     return this.cmsApiService.putUrl('/api/Users/' + user.id, data, {})
       .toPromise()
-      .then(User.extractData)
       .catch(this.handleError);
   }
 
+  getPicture(userId: String): Promise<any> {
+    let headers = new Headers();
+
+    headers.append('authorization', 'Bearer ' + localStorage.getItem('id_token'));
+    headers.append('Access-Control-Allow-Origin', '*');
+  
+    const url = 'http://docker-hip.cs.upb.de:5000/api/Users/' + userId + '/picture';
```

Comment:
There was an issue with this cmsApiSrvice url. If I changed it to this, then it does not take this new "multipart/form-data" header. We had looked into this issue long time back, and it wasn't working so we decided to use this hardcod url